libs/Microsoft.MixedReality.WebRTC/Interop/InteropUtils.cs

```diff
@@ -135,7 +135,7 @@ public static void ReleaseWrapperRef(IntPtr wrapperRef)
         }
 
         [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Ansi)]
-        internal struct SdpFilter
+        public struct SdpFilter
```

Comment:
This shouldn't need to be public.

Reply:
If I try to set this to internal I get this error on compiling:
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0051	Inconsistent accessibility: parameter type 'Utils.SdpFilter' is less accessible than method 'Utils.SdpForceCodecs(string, Utils.SdpFilter, Utils.SdpFilter, StringBuilder, ref ulong)'	Microsoft.MixedReality.WebRTC	C:\mrwebrtc\libs\Microsoft.MixedReality.WebRTC\Interop\InteropUtils.cs	170	Active
```

But I notice it is like this in master? Not sure if I am missing something, or need to set some .NET setting differently, but it appears to me like the way it is in master wouldn't compile.

Reply:
I see what's happening, `SdpForceCodecs` is declared as public but `Utils` is an internal class on master, so the method is actually internal. Here you have made `Utils` a public class so `SdpForceCodecs` is considered public, and the compiler wants also `SdpFilter` to be public.

AFAICS you only need `Utils` to be public because `ThrowOnErrorCode` is used in `NativeVideo`, right? I think the correct way to fix this is to leave `Utils` internal and have the native renderer library define its own error codes. Anyway at the moment it seems that all the methods are always returning `kSuccess` so I'd say just comment the `ThrowOnErrorCode` calls and revert this file here.

Reply:
`NativeVideo` still has a few calls to `ThrowOnErrorCode` that fail to compile now.